### PR TITLE
Sets language env variable when running shell

### PR DESCRIPTION
**Pull request by catenacyber**
Otherwise when I run
```
python infra/helper.py shell project
compile
```
I get the following error : `/usr/local/bin/compile: line 25: FUZZING_LANGUAGE: unbound variable`

### CODE REVIEW

**Comment by catenacyber** [comment]
CI says I crossed the limit : Too many lines in module (1003/1000)

File: infra/helper.py [hunk 1]
```diff
@@ -961,6 +961,10 @@ def shell(args):
   if args.e:
     env += args.e
 
+  project_language = _get_project_language(args.project_name)
+  if project_language:
+    env.append('FUZZING_LANGUAGE=' + project_language)
```
Comment (inferno-chromium) [inline infra/helper.py:966]: Project language will be always there, so add it in initial env list around line 958

**Review by inferno-chromium — CHANGES_REQUESTED** [review]

File: infra/helper.py (hunk 1 above)
Comment (catenacyber) [reply infra/helper.py]: I you are sure it is always there :-)

**Review by inferno-chromium — APPROVED** [review]

File: infra/helper.py (hunk 1 above)
Comment (inferno-chromium) [reply infra/helper.py]: Yes we require it now in project.yaml, also in presubmit check.

**Pull request merged**